lms/djangoapps/django_comment_client/management/commands/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
+Tests for management command line tools
+'''
+__author__ = 'azimpradhan'
```

Comment:
By convention, we don't add metadata to _init__.py We usually leave
these files empty, unless otherwise required.